config/initializers/validation_regex.rb

```diff
@@ -0,0 +1 @@
+EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/ix
```

Comment:
This can probably go in the flag model if we're not using it anywhere else?

Reply:
Sure. Good idea. I usually dump all validators in it's own initializer in case we have to use it again in the future.